diagnostic.md

```diff
@@ -9,6 +9,17 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+Boolean : true
+Null : Null
+Undefined : let red = ;
+String : ('cat', 'dog',)
```

Comment:
'cat' and 'dog' are individually examples of string, but the expression you wrote is not.